plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -2191,7 +2191,8 @@ public String getTitleToolTip() {
             .append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_path, scriptPath))
             .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_connecton, dataSourceContainer.getName()))
             .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_type, dataSourceContainer.getDriver().getFullName()))
-            .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_url, dataSourceContainer.getConnectionConfiguration().getUrl()));
+            .append("\n").append(NLS.bind(SQLEditorMessages.sql_editor_title_tooltip_url, dataSourceContainer.getConnectionConfiguration().getUrl()))
+            .append("\n").append(NLS.bind(SQLEditorMessages.sql_sql_editor_title_tooltip_project, dataSourceContainer.getFolder().getName()));
```

Comment:
A data source folder is not the same as a project it's contained in.

Use `DBPDataSourceContainer#getProject`